backend/apps/events/views.py

```
@@ -103,8 +103,10 @@ def _create_event_chat_group(
             )
 
         return chat_group, None
-    except Exception as e:
-        return None, str(e)
+    except IntegrityError:
+        return None, "Este evento ya tiene un chat asociado o el nombre del chat está duplicado."
+    except Exception:
```

Ruff (0.15.9)

[warning] Do not catch blind exception: `Exception`

(BLE001)